### ISSUE TITLE
Change uptime format in podman info from seconds to something like days, hours ?

### ISSUE THREAD

**Issue by sunilcio** [projectatomic/libpod#353]
`podman info` displays uptime in seconds. It would be better if the format is changed to days, hours.
```
  kernel: 4.14.18-300.fc27.x86_64
  os: linux
  uptime: "31692.03"
```
```
# rpm -qa | grep -i podman
podman-0.2.1-1.git3723cec.fc27.x86_64
```

**Comment by mheon** [comment]
Opened #355 to fix

**Comment by mheon** [comment]
#355 is merged, but @TomSweeneyRedHat has expressed a desire to further improve the output, so I'll leave this open for now.

**Comment by mheon** [comment]
Closed by #397
